Cartfile

```diff
@@ -1 +1 @@
-github "mapbox/MapboxDirections.swift" ~> 0.16
+github "mapbox/MapboxDirections.swift" ~> 2.0
```

Comment:
This repository has moved to `mapbox/mapbox-directions-swift`. The old name still works thanks to a redirect, but changing it to the new name ensures compatibility with other Mapbox libraries that use it.

```suggestion
github "mapbox/mapbox-directions-swift" ~> 2.0
```